src/StaticArrays.jl

```diff
@@ -21,6 +21,19 @@ import LinearAlgebra: transpose, adjoint, dot, eigvals, eigen, lyap, tr,
                       normalize!, Eigen, det, logdet, logabsdet, cross, diff, qr, \
 using LinearAlgebra: checksquare
 
+# StaticArraysCore imports
+# there is intentionally no "using StaticArraysCore" to not take all symbols exported
+# from StaticArraysCore to make transitioning definitions to StaticArraysCore easier.
+using StaticArraysCore: StaticArraysCore, StaticArray, StaticScalar, StaticVector,
+                        StaticMatrix, StaticVecOrMat, tuple_length, tuple_prod,
+                        tuple_minimum, size_to_tuple, require_one_based_indexing
+import StaticArraysCore: SArray, SVector, SMatrix
+import StaticArraysCore: MArray, MVector, MMatrix
+import StaticArraysCore: SizedArray, SizedVector, SizedMatrix
+import StaticArraysCore: check_array_parameters, convert_ntuple
+
+# end of StaticArraysCore imports
```

Comment:
Move the (re)-exports below to this block too maybe?

Reply:
Sure, done.